extensions/systemd/ConsumeJournald.cpp

```diff
@@ -23,6 +23,7 @@
 #include "date/date.h"
 #include "spdlog/spdlog.h"  // TODO(szaszm): make fmt directly available
 #include "utils/GeneralUtils.h"
+#include "utils/OptionalUtils.h"
```

Comment:
This shouldn't be needed.

Reply:
It's needed for utils::map, utils::flatMap and utils::orElse. Those are still our own and all of them are used in this file.

Reply:
I see, thanks!